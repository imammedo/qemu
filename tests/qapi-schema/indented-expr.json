##
# @eins:
##
{ 'command' : 'eins' }
##
# @zwei:
##
 { 'command' : 'zwei' }
