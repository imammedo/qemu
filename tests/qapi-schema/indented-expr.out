enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
command eins None -> None
   gen=True success_response=True boxed=False, runstates=None
object q_empty
command zwei None -> None
   gen=True success_response=True boxed=False, runstates=None
